src/python/pants/backend/jvm/tasks/scalafix.py

```diff
@@ -28,6 +28,8 @@ def register_options(cls, register):
              help='The config file to use (in HOCON format).')
     register('--rules', default='ProcedureSyntax', type=str, fingerprint=True,
              help='The `rules` arg to scalafix: generally a name like `ProcedureSyntax`.')
+    register('--scala-version', default='2.12.10', type=str, fingerprint=True,
```

Comment:
The default value is hardcoded to 2.12.10, is there a way to get Scala version from pants.ini or other build configuration?

Reply:
The major version is available on the `ScalaPlatform` subystem via something like `ScalaPlatform.global_instance().version`... the minor version is harder to find: would need to parse it from the relevant classpath if the classpath has been specified manually. Something like:
https://github.com/pantsbuild/pants/blob/2bdb2233b29a95fe04b39a15c79fdb971ba935bb/src/python/pants/backend/jvm/subsystems/zinc.py#L161-L170